b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/createRoleButton.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import { Button } from "rsuite";
+
+/**
+ * 
+ * @param prop - `session`
+ * 
+ * @returns - create role button
+ */
+export default function CreateRoleButton() {
+
+    return (
+        <Button
+            appearance="primary"
+            size="lg"
+            onClick={ ()=>{} }>
```

Comment:
Why is this an empty function?

Reply:
was this intended?

Reply:
This was fixed when the functionality of this file moved to a different location https://github.com/Achintha444/samples-is/commit/28b207622f0bf667fffe757611fbb7a8983a8429#diff-ae1fe67e81a5d047523b231d7112e52f5c53467b16d8500212e19f7fe856e144